site/tailwind.config.js

```diff
@@ -32,6 +32,7 @@ module.exports = {
 					primary: "hsl(var(--surface-primary))",
 					secondary: "hsl(var(--surface-secondary))",
 					tertiary: "hsl(var(--surface-tertiary))",
+					quaternary: "hsl(var(--surface-quaternary))",
```

Comment:
I was already against `primary`, `secondary` and `tertiary` as being three bad names, but now we're adding `quaternary`? this is such a horribly meaningless name